app/middleware.js

```diff
@@ -0,0 +1,122 @@
+const log = require('bole')('middleware');
```

Comment:
I'd rename this module to `middlewares`.

Reply:
This is the logger name, not a module. Log messages with be prefixed with (for example) `debug middleware: some log message`